samples/nrf9160/http_update/application_update/README.rst

```diff
@@ -50,30 +60,6 @@ Specifying the image files
 Before building the sample, you must specify where the image files are located.
 If you do not want to host the files yourself, you can upload them to a public S3 bucket on Amazon Web Services (AWS).
 
-Two options configure the names of the updated image files:
-
-- ``CONFIG_DOWNLOAD_FILE_V1`` - it configures the file name of the *version 1* of the update image.
-- ``CONFIG_DOWNLOAD_FILE_V2`` - it configures the file name of the *version 2* of the update image.
-
-To set these Kconfig options, follow these steps:
-
-.. tabs::
-
-   .. group-tab:: From |SES|
-
-      1. Select :guilabel:`Project` > :guilabel:`Configure nRF Connect SDK Project`.
-      #. Navigate to :guilabel:`HTTP application update sample` and specify both the download hostname (``CONFIG_DOWNLOAD_HOST``) and the names of the files that have to be downloaded (``CONFIG_DOWNLOAD_FILE_V1`` and ``CONFIG_DOWNLOAD_FILE_V2``).
-      #. Click :guilabel:`Configure` to save the configuration.
-
-   .. group-tab:: From the :file:`prj.conf` configuration file
-
-      1. Locate your :file:`prj.conf` project configuration file.
-      #. Change the ``CONFIG_DOWNLOAD_HOST`` option to indicate the hostname of the server where the image file is located (for example ``website.net``).
-      #. Change the ``CONFIG_DOWNLOAD_FILE_V1`` and ``CONFIG_DOWNLOAD_FILE_V2`` options to indicate the names and paths of the image files (for example ``download/app_update.bin``).
-
-      You can then use the :file:`prj.conf` project configuration file in your builds from the command line or using the *nRF Connect for Visual Studio Code* extension.
-      See the `Building and running`_ section for more information.
```

Comment:
Should we save some of this as it is not about SEGGER? I know we have the two Kconfig options mentioned above under Configuration, but not the CONFIG_DOWNLOAD_HOST one.

Reply:
Some of this needs to be kept, and some needs to be updated to map to VSC.

Reply:
My initial thought was for this sample, we set the Kconfig option in the same manner we do for other samples and in that case,  we direct the user to (line 41) http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_modifying.html#configure-application 
Hence, I removed both the SES and :file:`prj.conf` instructions.
If that is not the case, I could retain the :file:`prj.conf` instructions and for VSC could redirect to https://nrfconnect.github.io/vscode-nrf-connect/connect/build_app.html#multiple-build-configurations
 where we set the configurations ?

Reply:
I like your idea, @divipillai . Thoughts, @hakonfam , @peknis ?

Reply:
Fine with me.

Reply:
Makes sense.